components/core/src/error.rs

```diff
@@ -429,6 +432,7 @@ impl error::Error for Error {
                 "Service Bind strings must be in name:service_group format (example \
                  cache:redis.cache@organization)."
             }
+            Error::InvalidGroupName(_) => "Service group name must not contain a '.'",
```

Comment:
If this error is *only* for having a `.` where there shouldn't be, I think a different name would be appropriate. If it's intended to be more general, why don't we mirror the approach of `fmt` above:
```rust
format!("Invalid group: {}.", e),
```
?

Reply:
renamed to `DotInGroupName`